units/XNB0304/XNB0304_script.lua

```diff
@@ -20,26 +21,37 @@ XNB0304 = Class(NSCUFactoryUnit) {
         self.TractorBeamBag = TrashBag()
     end,
 
+    ---@param self XNB0304
     OnDestroy = function(self)
         self:DestroyTractorBeamFx()
         NSCUFactoryUnit.OnDestroy(self)
     end,
 
+    ---@param self XNB0304
+    ---@param instigator Unit
+    ---@param type string
+    ---@param overkillRatio number
     OnKilled = function(self, instigator, type, overkillRatio)
         self:DestroyTractorBeamFx()
         NSCUFactoryUnit.OnKilled(self, instigator, type, overkillRatio)
     end,
 
+    ---@param self XNB0304
+    ---@param builder Unit
+    ---@param layer string
```

Comment:
Jip made an upgrade to the layer annotations a while back
```suggestion
    ---@param layer Layer
```